lms/envs/common.py

```diff
@@ -2452,6 +2452,11 @@ def _make_locale_paths(settings):
     'google': '#'
 }
 
+################# Learner Profile ##################
+
+# TODO (DEPR-TODO)
+PROFILE_MICROFRONTEND_URL_BASE = "http://localhost:18000/profile-spa/"
```

Comment:
This default should probably be something other than localhost:18000 to avoid confusion.

Reply:
Good point.  Updated.